driverconfig/unity_v230_v121.json

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
Please remove json files related to 2.0.0

Reply:
taken care